hera_mc/autocorrelations.py

```diff
@@ -31,46 +33,97 @@ class _MeasurementTypes(object):
     median = 0
     "The median value across the whole autocorrelation spectrum."
 
-    names = ['median']
+    names = ["median"]
     "A list of textual names corresponding to each value."
 
 
 MeasurementTypes = _MeasurementTypes()
 
 
-class Autocorrelations(MCDeclarativeBase):
-    """A table logging antenna autocorrelations.
-
+class HeraAuto(MCDeclarativeBase):
+    """
+    Definition of median antenna autocorrelation table of hera antennas.
+
+    Attributes
+    ----------
+    time : BigInteger Column
+        The time in GPS seconds of the observation, floored as an int. Part of primary_key
+    antenna_number : Integer Column
+        Antenna number. Part of primary_key.
+    antenna_feed_pol : String Column
+        Feed polarization, either 'e' or 'n'. Part of primary_key.
+    measurement_type : SmallInt Column
+        The type of measurement; see MeasurementTypes enumeration.
+        Cannot be None.
```

Comment:
Why do you want to use an int here rather than just a string? We can impose a constraint that the name is in the allowed values.

Reply:
This was also a carry over from the first autocorrelations table and I did not know about the string constraints in the databases.

Reply:
or do you mean to just use python checking in the `create` method like https://github.com/HERA-Team/hera_mc/blob/7b928319c07ddde14bf1c6033ab7096605108607/hera_mc/correlator.py#L116-L119